### PR TITLE
PORT 10426 New Relic Add Cloud Resources

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

Added new guide for new relic's additon of cloud resources
## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- Guide (`/guides/all/ingest-cloud-resources-using-newrelic`)

### CODE REVIEW

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+## Prerequisites
+
+This guide assumes the following:
+
+- You have a Port account and have completed the [onboarding process](https://docs.getport.io/quickstart).
+- You have [installed and set up Port's New Relic integration](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/apm-alerting/newrelic).
+- You have entities from cloud providers configured in New Relic. See [New Relic's documentation](https://docs.newrelic.com/docs/infrastructure/) for details on setting up cloud integrations.
+
+## Ingesting Cloud Resources into Port
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:23]:
```suggestion
## Ingesting cloud resources into Port
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+We will utilize the `entity` kind in Port's New Relic integration, which provides information on entities monitored in New Relic. Entities are ingested based on their respective `infrastructureIntegrationType`.
+
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:26]:
```suggestion
We will utilize the `entity` kind in Port's New Relic integration, which provides information on entities monitored in New Relic. Entities are ingested based on their respective infrastructure integration type.
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+To ingest cloud resources, we need to specify entities of certain `infrastructureIntegrationType` values corresponding to your cloud resources.
+
+<details>
+<summary><b>Some cloud resources infrastructure integration types</b></summary>
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:30]:
```suggestion
<summary><b>Examples of cloud resources infrastructure integration types</b></summary>

```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+- `AZURE_VIRTUAL_MACHINE`
+- `AZURE_SQL_DATABASE`
+- `GCP_COMPUTE_INSTANCE`
+- `GCP_STORAGE_BUCKET`
+- `GCP_SQL_DATABASE_INSTANCE`
+
+</details>
+
+
+## Data Model Setup
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:46]:
```suggestion
## Data model setup
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+Follow the steps below to set your data model up for ingesting cloud resources from New Relic:
+
+1. Creating the Blueprint Configuration
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:49]:
```suggestion
1. Creating the blueprint configuration
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+    },
+    "required": []
+  },
+  "relations": {}
+}
+```
+
+</details>
+
+2. Configuring the Integration Mapping
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:109]:
```suggestion
2. Configuring the integration mapping
```

**Review by PeyGis — CHANGES_REQUESTED** [review]
looks good overall. left some suggestions

**Review by PeyGis — APPROVED** [review]

File: src/components/guides-section/consts.js [hunk 7]
```diff
@@ -804,4 +805,12 @@ export const availableGuides = [
         logos: ["Trivy", "GitHub"],
         link: "/guides/all/ingest-trivy-vulnerabilities-into-your-catalog"
     },
+    {
+        title: "Ingest cloud resources using New Relic",
+        description: "Ingest any cloud resources from your New Relic environment into Port",
+        tags: ["Engineering metrics", "New Relic"],
+        logos: ["NewRelic"],
```
Comment (hadar-co) [inline src/components/guides-section/consts.js:812]:
```suggestion
        logos: ["New Relic"],
```

File: src/components/guides-section/consts.js [hunk 8]
```diff
@@ -804,4 +805,12 @@ export const availableGuides = [
         logos: ["Trivy", "GitHub"],
         link: "/guides/all/ingest-trivy-vulnerabilities-into-your-catalog"
     },
+    {
+        title: "Ingest cloud resources using New Relic",
+        description: "Ingest any cloud resources from your New Relic environment into Port",
```
Comment (hadar-co) [inline src/components/guides-section/consts.js:810]:
```suggestion
        description: "Ingest cloud resources from your New Relic environment into Port",
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+            tags: .tag
+```
+
+</details>
+
+3. Resyncing Data
+
+After configuring the mapping, click on `Resync` to start ingesting your cloud resources from New Relic into Port.
+
+<img src="/img/guides/newRelicIngestedData.png" />
```
Comment (hadar-co) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:153]:
```suggestion
<img src="/img/guides/newRelicIngestedData.png" border='1px' />
```

File: docs/guides/all/ingest-cloud-resources-using-newrelic.md [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+  - `NGINX_SERVER`
+  - `MYSQL_DATABASE`
+  - `POSTGRESQL_DATABASE`
+  - `REDIS_INSTANCE`
+  - `DOCKER_CONTAINER`
+  - `KUBERNETES_CLUSTER`
+
+</details>
+
+:::note Follow up to date
```
Comment (hadar-co) [inline docs/guides/all/ingest-cloud-resources-using-newrelic.md:224]:
```suggestion
:::note Latest information
```

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**